subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[console]]
+== The Gradle Console
+
+NOTE: This feature is a work in progress.
+
+[[sec:console_overview]]
+=== Overview
+
+The console is the central user interface for any user executing Gradle from the command line. Gradle's console is optimized for rendering performance, showing only relevant information while at the same time providing a visually appealing feedback loop.
+
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>The Gradle console in action</title>
+    <imageobject>
+        <imagedata fileref="img/console-animation.gif"/>
```

Comment:
We should record an example of this that doesn't have a deprecation warning in it.  To work around it, we can just wrap the application of the jmh plugin in a `SingleMessageLogger.whileDisabled()`